lib/salus/scanners/yarn_audit.rb

```diff
@@ -10,7 +10,9 @@ class YarnAudit < NodeAudit
     # the command was previously 'yarn audit --json', which had memory allocation issues
     # see https://github.com/yarnpkg/yarn/issues/7404
     LEGACY_YARN_AUDIT_COMMAND = 'yarn audit --no-color'.freeze
-    LATEST_YARN_AUDIT_COMMAND = 'yarn npm audit --all --json'.freeze
+    LATEST_YARN_AUDIT_ALL_COMMAND = 'yarn npm audit --recursive --all --json'.freeze
```

Comment:
Can we add a config option for recursive?

Does yarn <3 support recursive?  If so we should mirror behavior between yarns (ideally both using the same config var to determine if the scan will be recursive)